dashboard/app/models/script.rb

```diff
@@ -1871,6 +1872,21 @@ def alternate_script(user)
     nil
   end
 
+  def summarize_for_assignment_dropdown
+    info = ScriptConstants.assignable_info(self)
+
+    [
+      id,
+      {
+        id: id,
+        name: I18n.t("data.script.name.#{info[:name]}.title", default: info[:name]),
```

Comment:
How is this different than `localized_title`?

Reply:
I don't think it is. I will update!